index.bs

```diff
@@ -1143,6 +1146,25 @@ The following constraints define when a [=/transaction=] can be
 
 </div>
 
+To <dfn export>cleanup Indexed Database transactions</dfn> for an
+[=event loop=], run the following steps for each [=/transaction=]
+with a matching [=transaction/cleanup event loop=]:
```

Comment:
Can you suggest text here? Just use "the event loop", or "the relevant event loop" or ...?